.gitignore

```diff
@@ -2,5 +2,6 @@
 
 ### Gradle ###
 .gradle/
+*.json
```

Comment:
Do not add .json to gitignore - a number of asset configurations are stored in .json files and may need to be modified (find another way to ignore loading/saving files)